### PR TITLE
Updated UG

### PR DESCRIPTION
**Pull request by raisa2010**
- The edits for everyone else's writing are in "<>". Please make relevant changes or delete the brackets if you think it's unnecessary.

- Once you're done, make a PR with ONLY the changes to your documentation (UG).

- Tag me if you want me to review.

- Merge to master. Once you're all done, I intend to pull the changes to this branch and commit so the edits should no longer be there.

- Let me know if you find any errors in formatting/typos etc since they can be easy to miss.

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -190,165 +186,154 @@ Returns any person having birthday of `17-07`
...
 
 `tag 1, 2, 3 t/friend t/family` +
-Results in all contacts in indices 1, 2 and 3 to be tagged with both [friend] and [family].
+Results in all contacts in indices 1, 2 and 3 being tagged with both [`friend`] and [`family`].
 
 __Constraints__
 
-****
-* Each index MUST be valid according to the current list of contacts.
+Note the following constraints when trying to add tags:
```
Comment (tpq95) [inline docs/UserGuide.adoc]: a plus sign in front does not break a line,
do you want the format to be a `+` in front of every line in constraint?

____TEST____
+Note this
+*format

versus

Note +
this +

Comment (raisa2010) [reply docs/UserGuide.adoc]: Nope, no plus signs in the actual formatting except wherever it's needed to break lines. That's probably a stray but it doesn't seem to show up in the actual document. You can delete it or leave it.

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -358,48 +343,46 @@ __Constraints__
...
-* Dates MUST be preceded by one of the prefixes.
-****
+Note the following constraints when adding tasks:
+
+* Dates *must* be preceded by one of the prefixes.
+* Invalid dates (such as 32 Jan) will not be flagged by Mobilize and will be replaced in a task card by an incorrect date (such as Feb 1 '48).
 
 ==== Finding tasks by description or deadline: `find`
+<add lead in line about filtering to interact with user>
+<find *tasks>
```
Comment (tpq95) [inline docs/UserGuide.adoc]: what do you mean by `*tasks`?

File: docs/UserGuide.adoc (hunk 1 above)
Comment (tpq95) [reply docs/UserGuide.adoc]: i see, then i will put + sign at end of line to break the sentence. but this is under the tag command, i am referencing it, so you may want to change it. thanks!

Comment (tpq95) [reply docs/UserGuide.adoc]: Sorry my bad, i mix up the GitHub changes with the code.

Comment (raisa2010) [reply docs/UserGuide.adoc]: no worries, easy to miss these things

**Pull request merged**